addon/components/leaflet-map.js

```diff
@@ -39,7 +40,9 @@ export default BaseLayer.extend(ParentMixin, {
   ],
 
   leafletProperties: [
-    'zoom:setZoom:zoomPanOptions', 'center:panTo:zoomPanOptions', 'maxBounds:setMaxBounds', 'bounds:fitBounds:fitBoundsOptions'
+    'zoom:setZoom:zoomPanOptions', 'minZoom:setMinZoom:zoomPanOptions', 'maxZoom:setMaxZoom:zoomPanOptions',
```

Comment:
Do we need the `:zoomPanOptions` part here for minZoom and maxZoom?
AFAIK, `setMinZoom` and `setMaxZoom` accept only one argument which is the zoom level itself.